css/nav.less

```diff
@@ -488,6 +488,24 @@ nav {
   }
 }
 
+.leftNavIsOpen #topNav .main-menu {
+  @media (max-width: (@main-menu-hide-breakpoint + @left-nav-width - 1)) {
+    display: none;
```

Comment:
this feels wrong to me -- it makes the menu jump between the left nav and the top nav when you toggle the left nav. will things still fit if you just remove this rule?